include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -367,12 +367,12 @@ namespace LinearAlgebra
        * This function is mainly for internal consistency checks and should
        * seldom be used when not in debug mode since it uses quite some time.
        */
-      virtual bool all_zero () const override;
+      bool all_zero () const override;
```

Comment:
I think that @bangerth wanted to use `virtual` when we did override a function. Not sure if we should still do it since we add `override` now